eo-runtime/src/test/java/org/eolang/EOintEOpowTest.java

```diff
@@ -51,4 +53,59 @@ public void powersNumber() {
         );
     }
 
+    @Test
+    public void zeroNonNegativePowers() {
+        // 0^0
+        MatcherAssert.assertThat(
```

Comment:
@nlchar it's better to have one assertion per test: https://softwareengineering.stackexchange.com/questions/7823/is-it-ok-to-have-multiple-asserts-in-a-single-unit-test

Reply:
@yegor256 thank you. Fixed that. I kept two asserts in `zeroToNegativePowerFails`, though. Because those test that:
1. The `error` object is returned and the object has the proper `msg` value.
2. The code does not datarize, so it would fail in real-life code, which is our goal. 

The QA you attached accepts this. I can change it if you like.